eventmesh-examples/build.gradle

```diff
@@ -25,7 +25,9 @@ dependencies {
     implementation('org.springframework.boot:spring-boot-starter') {
         exclude module: 'spring-boot-starter-logging'
     }
-    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation('org.springframework.boot:spring-boot-starter-web') {
+        exclude module: 'spring-boot-starter-logging'
+    }
```

Comment:
Actually `org.springframework.boot:spring-boot-starter` is included in `org.springframework.boot:spring-boot-starter-web` as is commented in https://github.com/apache/eventmesh/pull/4719/files#r1570135028.

`group` param is missing in the exclude statement.